src/lib/feature-finder.js

```diff
@@ -50,7 +53,7 @@ export default function(cucumberOptions) {
     });
   })
   .then((results) => {
-    return _.flattenDeep(results);
+    return {scenarios:_.flattenDeep(results), didDetectErrors: didDetectErrors};
```

Comment:
You can use es6 shorthand and just `return {scenarios: _flattenDeep(results), didDetectErrors};`